library/include/rocwmma/internal/layout_impl.hpp

```diff
@@ -851,23 +848,26 @@ namespace rocwmma
                 {
                     using OrthoLayout
                         = ColInlineVW<BlockDim, BlockK, DataT, VectorWidth, MaxVectorWidth>;
-                    using MatrixCoordT = std::pair<int32_t, int32_t>;
+                    using MatrixCoordT = HIP_vector_type<uint32_t, 2>;
                 };
 
                 // Matrix coord offsets
                 __device__ static inline typename Traits::MatrixCoordT baseOffset()
                 {
-                    return std::swap(Traits::OrthoLayout::baseOffset());
```

Comment:
Recommend to implement a rocwmma::swap utility for the MatrixCoordT type, such that tedious expansion into .x and .y isn't required.
Manual expansion is higher risk for error introduction and should be consolidated into a swap utility